home-work14/test_for_human.py

```diff
@@ -0,0 +1,94 @@
+import pytest
+
+from human import Human
+
+
+def test_human_grow():
+    human = Human("John", 30, "male")
+    human.grow()
+    assert human.age == 31, "Age is not 30"
```

Comment:
We do not use magic numbers in Test cases. You should calculate expected result where it is possible, not hardoce.
In your case you know that grow() will increase age by one. So you can take age before growing and increment it by 1, in that case we will get correct expected result